docs/spec_draft.md

```diff
@@ -56,6 +56,12 @@ implementation-defined, as long as ops are evaluated before their uses. Possible
 execution orders of the above example program are `%1` → `%2` → `%3` → `return`
 or `%2` → `%1` → `%3` → `return`.
 
+### Floating-point Exceptions
+
+The execution envorinment of StableHLO does not maintain any exception state.
+Therefore, there is no attempt to create or preserve invalid operation,
+  division-by-zero, overflow, underflow, or inexact exceptions.
+
```

Comment:
How about making this more general and taking this chance to talk about errors? Not sure that's necessarily the right word to use, but let's iterate from there? For example:

## Errors

StableHLO programs are validated through an extensive set of constraints for individual ops, which rules out many classes of errors prior to run time. However, error conditions are still possible, e.g. through integer overflows, out-of-bounds accesses, etc. Unless explicitly called out, all these errors result in implementation-defined behavior. 

As an exception to this rule, floating-point exceptions in StableHLO programs have well-defined behavior. Operations which result in exceptions defined by the IEEE-754 standard (invalid operation, division-by-zero, overflow, underflow, or inexact exceptions) produce default results.

Reply:
Thanks! updated the text with some changes.